scripts.js

```diff
@@ -6,21 +6,68 @@ let oneDay = 1000*60*60*24;
 let daysLeft = Math.round((endOfChallenge - currentDate)/oneDay);
 let daysLeftDisplay = document.querySelector('#daysLeftDisplay');
 let bookFrequencyDisplay = document.querySelector('#bookFrequencyDisplay');
-console.log(currentDate);
+let frequencyDisplayText = document.querySelector('#frequencyDisplayText');
+
+const writeToLocalStorage = (goal, read) => {
+  if (Storage !== undefined){
+    localStorage.setItem("goal", goal);
+    localStorage.setItem("read", read);
+  }
+}
+
+const getFromLocalStorage = (key) => {
+  if (Storage !== undefined){
+    const storedVal = localStorage.getItem(key);
+    if (storedVal != null){
+      return storedVal;
+    }else{
+      return "";
+    }
+  }
+  return "";
+}
+
+const resetFrequencyDisplayTextParagraph = () => {
```

Comment:
This function would reset the paragraph tag to its original structure, by using the createElement and createTextNode functions of JS